### DISCUSSION TITLE
Nested css with component style not working

### DISCUSSION THREAD

**Discussion by Akshay-pal** [vercel/next.js#13349] [Help]
# Bug report

## Describe the bug

I am using restrape tabs and I want different style on which tab that has only **.active** class but scoped style not working, I am getting stuck in this situation.Here is simple scss code for active tab.

.nav_item{
        width: 50%;
        a.active{
                color: #ffffff;
                background-color: #467c9c !important;
                border-color: #dee2e6 #dee2e6 #fff;
                padding: 15px 15px;
                text-align: center;
        }
    }

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Simply use reactstrap tabs
2. Try to apply css only on active tab

## Expected behavior

It should work with active tab that must have different styles that was added on scoped style.

## Additional context

Pls suggest me what should I do. I am trying to resolve since 2-3 days but still not got any solution.

**Comment by jamesmosier** [comment]
Hi @Akshay-pal could you provide a repository that shows what you are trying to do?
